docs/src/content/docs/fr/components/asides.mdx

```diff
@@ -0,0 +1,158 @@
+---
+title: Encarts
+description: Apprenez à utiliser les encarts dans Starlight pour afficher des informations secondaires à côté du contenu principal d'une page.
+---
+
+import { Aside } from '@astrojs/starlight/components';
+
+Pour afficher des informations secondaires à côté du contenu principal d'une page, utilisez le composant `<Aside>`.
+
+import Preview from '~/components/component-preview.astro';
+
+<Preview>
+
+<Aside slot="aperçu">
```

Comment:
```suggestion
<Aside slot="preview">
```

Looks like you translated some slot names and kept some untranslated. They should never be translated otherwise this breaks the rendering of the preview as the component expect a slot with this specific name.

Reply:
Oh was a mistake